README.md

```diff
@@ -29,6 +29,8 @@ We use pre-commit to make sure the code is consistently formatted. To make sure
 
 ### Testing
 
+> Some tests, i.e. test_ray_details, test_cluster_status, and test_wait_ready require an OpenShift cluster with the [Codeflare Stack](https://github.com/opendatahub-io/distributed-workloads/blob/main/Quick-Start.md#quick-start-guide-for-distributed-workflows-with-the-codeflare-stack) installed.
```

Comment:
I don't think this is true. Unless something has changed recently, all of the unit tests can be run locally **without** an OpenShift Cluster. 

We use mocker to fake any required OC responses.